common/important.rkt

```diff
@@ -0,0 +1,3 @@
+#lang racket
```

Comment:
LOL

can you move this into the scripts folder?

and maybe we could have an racket-themed easter egg on the site - maybe like something comes up when you type in "racket"?  🥚 🤔

Reply:
:o